build/Build.cs

```diff
@@ -96,10 +96,25 @@ partial class Build : NukeBuild
         //.OnlyWhenStatic(() => InvokedTargets.Contains(nameof(RunChangelog)))
         .Executes(() =>
         {
-            FinalizeChangelog(ChangelogFile, GitVersion.SemVer, GitRepository);
+            // GitVersion.SemVer appends the branch name to the version numer (this is good for pre-releases)
+            // If you are executing this under a branch that is not beta or alpha - that is final release branch
+            // you can update the switch block to reflect your release branch name
+            var vNext = string.Empty;
+            var branch = GitVersion.BranchName;
+            switch (branch)
```

Comment:
`GitVersion.SemVer` appends the branch name to the version number (this is good for pre-releases)
but for final releases `GitVersion.MajorMinorPatch` is the way to go!